File: tests/fixtures/sample.py
def calculate_metrics(data: dict) -> float:
    """
    Calculates important business metrics.
    """
    total = sum(data.values())
    return float(total) / 100.0

class Processor:
    """
    Main processing class.
    """
    def __init__(self):
        self.ready = True
        
    async def process_async(self, item):
        # async processing
        pass

File: tests/fixtures/sample.js
function calculateMetrics(data) {
    // Calculates important business metrics.
    const values = Object.values(data);
    const total = values.reduce((a, b) => a + b, 0);
    return total / 100.0;
}

class Processor {
    constructor() {
        this.ready = true;
    }
    
    async processAsync(item) {
        // async processing
        const res = await fetch(item);
        return res.json();
    }
}

const helper = (a, b) => {
    return a + b;
};
